### ISSUE TITLE
NPE (EMF-NiCad)

### ISSUE THREAD

**Issue by dmazinanian** [tsantalis/JDeodorant#49]
Clone # 173 (Group 173, Pair 60-116): org.eclipse.emf.ecore.change.impl.ResourceChangeImpl#getListChanges (row 467) and org.eclipse.emf.ecore.resource.impl.ResourceImpl#getContents (row 523)

```
java.lang.NullPointerException
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.extractClone(ExtractCloneRefactoring.java:1338)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.apply(ExtractCloneRefactoring.java:340)
    at gr.uom.java.jdeodorant.refactoring.manipulators.ExtractCloneRefactoring.checkFinalConditions(ExtractCloneRefactoring.java:5031)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.testRefactoring(Application.java:669)
    at ca.concordia.jdeodorant.eclipse.commandline.Application.start(Application.java:185)
    at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:196)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:134)
    at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:388)
    at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:243)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:498)
    at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:673)
    at org.eclipse.equinox.launcher.Main.basicRun(Main.java:610)
    at org.eclipse.equinox.launcher.Main.run(Main.java:1519)
    at org.eclipse.equinox.launcher.Main.main(Main.java:1492)
```

**Comment by dmazinanian** [comment]
Possibly the same issue in EMF-CloneDR, Group 86, Pair 2-3

**Comment by tsantalis** [comment]
Fixed in commit [2373ab98f5f59fc9ae7431b2f078f9a1f40b1bad](https://github.com/tsantalis/JDeodorant/commit/2373ab98f5f59fc9ae7431b2f078f9a1f40b1bad)
